Clients/GUI/Shared/Currency/CurrencyContainer.razor

```diff
@@ -0,0 +1,43 @@
+﻿@using System.Net.Http
+
+@if (Currencies != null)
+{
+    <p class="badge-dark" style="margin-bottom:1px">CURRENCY RATE</p>
+    @if (CurrencyList == null)
+    {
+        @foreach (var cur in Currencies)
+        {
+            <div class="text-center container-fluid" style="white-space:nowrap">
+                <span class="item i-body">@cur.Id</span>
+                <span class="item i-body">@cur.Value</span>
+            </div>
+        }
+    }
+    else
+    {
+        @foreach (var cur in Currencies)
```

Comment:
Probably, we have to first find all currencies that we want to write? 

And also, is all this have to be at GUI ? Probably, we have to change Backend part.

Reply:
I do not think so

Reply:
Okay. Can you please explain?

Reply:
explain your point of view

Reply:
I think we had a suggestion in conversation in skype